Update Icons to v1.11.3

alexkazik opened the pull request:
Updates the included/referenced bootstrap-icons version.

Please be aware that this is a breaking change for the users if the cdn is not used (as there is a version number in the path to the file). (I have an idea to avoid this in the future, I'll update the PR later.)

Changes to the generated icons:
```diff
--- bootstrap_icons_generated-v1.10.5.rs	2024-04-06 11:03:32
+++ bootstrap_icons_generated-v1.11.3.rs	2024-04-06 11:05:37
@@ -56,7 +56,10 @@
     pub const ALIGN_START: BI = BI("<i class=\"bi bi-align-start\"></i>");
     pub const ALIGN_TOP: BI = BI("<i class=\"bi bi-align-top\"></i>");
     pub const ALIPAY: BI = BI("<i class=\"bi bi-alipay\"></i>");
+    pub const ALPHABET: BI = BI("<i class=\"bi bi-alphabet\"></i>");
+    pub const ALPHABET_UPPERCASE: BI = BI("<i class=\"bi bi-alphabet-uppercase\"></i>");
     pub const ALT: BI = BI("<i class=\"bi bi-alt\"></i>");
+    pub const AMAZON: BI = BI("<i class=\"bi bi-amazon\"></i>");
     pub const AMD: BI = BI("<i class=\"bi bi-amd\"></i>");
     pub const ANDROID: BI = BI("<i class=\"bi bi-android\"></i>");
     pub const ANDROID2: BI = BI("<i class=\"bi bi-android2\"></i>");
@@ -126,12 +129,16 @@
     pub const ARROW_UP_SHORT: BI = BI("<i class=\"bi bi-arrow-up-short\"></i>");
     pub const ARROW_UP_SQUARE: BI = BI("<i class=\"bi bi-arrow-up-square\"></i>");
     pub const ARROW_UP_SQUARE_FILL: BI = BI("<i class=\"bi bi-arrow-up-square-fill\"></i>");
+    pub const ARROWS: BI = BI("<i class=\"bi bi-arrows\"></i>");
     pub const ARROWS_ANGLE_CONTRACT: BI = BI("<i class=\"bi bi-arrows-angle-contract\"></i>");
     pub const ARROWS_ANGLE_EXPAND: BI = BI("<i class=\"bi bi-arrows-angle-expand\"></i>");
     pub const ARROWS_COLLAPSE: BI = BI("<i class=\"bi bi-arrows-collapse\"></i>");
+    pub const ARROWS_COLLAPSE_VERTICAL: BI = BI("<i class=\"bi bi-arrows-collapse-vertical\"></i>");
     pub const ARROWS_EXPAND: BI = BI("<i class=\"bi bi-arrows-expand\"></i>");
+    pub const ARROWS_EXPAND_VERTICAL: BI = BI("<i class=\"bi bi-arrows-expand-vertical\"></i>");
     pub const ARROWS_FULLSCREEN: BI = BI("<i class=\"bi bi-arrows-fullscreen\"></i>");
     pub const ARROWS_MOVE: BI = BI("<i class=\"bi bi-arrows-move\"></i>");
+    pub const ARROWS_VERTICAL: BI = BI("<i class=\"bi bi-arrows-vertical\"></i>");
     pub const ASPECT_RATIO: BI = BI("<i class=\"bi bi-aspect-ratio\"></i>");
     pub const ASPECT_RATIO_FILL: BI = BI("<i class=\"bi bi-aspect-ratio-fill\"></i>");
     pub const ASTERISK: BI = BI("<i class=\"bi bi-asterisk\"></i>");
@@ -139,6 +146,14 @@
     pub const AWARD: BI = BI("<i class=\"bi bi-award\"></i>");
     pub const AWARD_FILL: BI = BI("<i class=\"bi bi-award-fill\"></i>");
     pub const BACK: BI = BI("<i class=\"bi bi-back\"></i>");
+    pub const BACKPACK: BI = BI("<i class=\"bi bi-backpack\"></i>");
+    pub const BACKPACK_FILL: BI = BI("<i class=\"bi bi-backpack-fill\"></i>");
+    pub const BACKPACK2: BI = BI("<i class=\"bi bi-backpack2\"></i>");
+    pub const BACKPACK2_FILL: BI = BI("<i class=\"bi bi-backpack2-fill\"></i>");
+    pub const BACKPACK3: BI = BI("<i class=\"bi bi-backpack3\"></i>");
+    pub const BACKPACK3_FILL: BI = BI("<i class=\"bi bi-backpack3-fill\"></i>");
+    pub const BACKPACK4: BI = BI("<i class=\"bi bi-backpack4\"></i>");
+    pub const BACKPACK4_FILL: BI = BI("<i class=\"bi bi-backpack4-fill\"></i>");
     pub const BACKSPACE: BI = BI("<i class=\"bi bi-backspace\"></i>");
     pub const BACKSPACE_FILL: BI = BI("<i class=\"bi bi-backspace-fill\"></i>");
     pub const BACKSPACE_REVERSE: BI = BI("<i class=\"bi bi-backspace-reverse\"></i>");
@@ -183,6 +198,8 @@
     pub const BALLOON_FILL: BI = BI("<i class=\"bi bi-balloon-fill\"></i>");
     pub const BALLOON_HEART: BI = BI("<i class=\"bi bi-balloon-heart\"></i>");
     pub const BALLOON_HEART_FILL: BI = BI("<i class=\"bi bi-balloon-heart-fill\"></i>");
+    pub const BAN: BI = BI("<i class=\"bi bi-ban\"></i>");
+    pub const BAN_FILL: BI = BI("<i class=\"bi bi-ban-fill\"></i>");
     pub const BANDAID: BI = BI("<i class=\"bi bi-bandaid\"></i>");
     pub const BANDAID_FILL: BI = BI("<i class=\"bi bi-bandaid-fill\"></i>");
     pub const BANK: BI = BI("<i class=\"bi bi-bank\"></i>");
@@ -210,6 +227,7 @@
     pub const BEZIER: BI = BI("<i class=\"bi bi-bezier\"></i>");
     pub const BEZIER2: BI = BI("<i class=\"bi bi-bezier2\"></i>");
     pub const BICYCLE: BI = BI("<i class=\"bi bi-bicycle\"></i>");
+    pub const BING: BI = BI("<i class=\"bi bi-bing\"></i>");
     pub const BINOCULARS: BI = BI("<i class=\"bi bi-binoculars\"></i>");
     pub const BINOCULARS_FILL: BI = BI("<i class=\"bi bi-binoculars-fill\"></i>");
     pub const BLOCKQUOTE_LEFT: BI = BI("<i class=\"bi bi-blockquote-left\"></i>");
@@ -293,6 +311,7 @@
     pub const BRIGHTNESS_HIGH_FILL: BI = BI("<i class=\"bi bi-brightness-high-fill\"></i>");
     pub const BRIGHTNESS_LOW: BI = BI("<i class=\"bi bi-brightness-low\"></i>");
     pub const BRIGHTNESS_LOW_FILL: BI = BI("<i class=\"bi bi-brightness-low-fill\"></i>");
+    pub const BRILLIANCE: BI = BI("<i class=\"bi bi-brilliance\"></i>");
     pub const BROADCAST: BI = BI("<i class=\"bi bi-broadcast\"></i>");
     pub const BROADCAST_PIN: BI = BI("<i class=\"bi bi-broadcast-pin\"></i>");
     pub const BROWSER_CHROME: BI = BI("<i class=\"bi bi-browser-chrome\"></i>");
@@ -336,6 +355,10 @@
     pub const C_CIRCLE_FILL: BI = BI("<i class=\"bi bi-c-circle-fill\"></i>");
     pub const C_SQUARE: BI = BI("<i class=\"bi bi-c-square\"></i>");
     pub const C_SQUARE_FILL: BI = BI("<i class=\"bi bi-c-square-fill\"></i>");
+    pub const CAKE: BI = BI("<i class=\"bi bi-cake\"></i>");
+    pub const CAKE_FILL: BI = BI("<i class=\"bi bi-cake-fill\"></i>");
+    pub const CAKE2: BI = BI("<i class=\"bi bi-cake2\"></i>");
+    pub const CAKE2_FILL: BI = BI("<i class=\"bi bi-cake2-fill\"></i>");
     pub const CALCULATOR: BI = BI("<i class=\"bi bi-calculator\"></i>");
     pub const CALCULATOR_FILL: BI = BI("<i class=\"bi bi-calculator-fill\"></i>");
     pub const CALENDAR: BI = BI("<i class=\"bi bi-calendar\"></i>");
@@ -632,6 +655,8 @@
     pub const CONE: BI = BI("<i class=\"bi bi-cone\"></i>");
     pub const CONE_STRIPED: BI = BI("<i class=\"bi bi-cone-striped\"></i>");
     pub const CONTROLLER: BI = BI("<i class=\"bi bi-controller\"></i>");
+    pub const COOKIE: BI = BI("<i class=\"bi bi-cookie\"></i>");
+    pub const COPY: BI = BI("<i class=\"bi bi-copy\"></i>");
     pub const CPU: BI = BI("<i class=\"bi bi-cpu\"></i>");
     pub const CPU_FILL: BI = BI("<i class=\"bi bi-cpu-fill\"></i>");
     pub const CREDIT_CARD: BI = BI("<i class=\"bi bi-credit-card\"></i>");
@@ -641,6 +666,8 @@
     pub const CREDIT_CARD_2_FRONT_FILL: BI = BI("<i class=\"bi bi-credit-card-2-front-fill\"></i>");
     pub const CREDIT_CARD_FILL: BI = BI("<i class=\"bi bi-credit-card-fill\"></i>");
     pub const CROP: BI = BI("<i class=\"bi bi-crop\"></i>");
+    pub const CROSSHAIR: BI = BI("<i class=\"bi bi-crosshair\"></i>");
+    pub const CROSSHAIR2: BI = BI("<i class=\"bi bi-crosshair2\"></i>");
     pub const CUP: BI = BI("<i class=\"bi bi-cup\"></i>");
     pub const CUP_FILL: BI = BI("<i class=\"bi bi-cup-fill\"></i>");
     pub const CUP_HOT: BI = BI("<i class=\"bi bi-cup-hot\"></i>");
@@ -731,6 +758,8 @@
     pub const DROPLET: BI = BI("<i class=\"bi bi-droplet\"></i>");
     pub const DROPLET_FILL: BI = BI("<i class=\"bi bi-droplet-fill\"></i>");
     pub const DROPLET_HALF: BI = BI("<i class=\"bi bi-droplet-half\"></i>");
+    pub const DUFFLE: BI = BI("<i class=\"bi bi-duffle\"></i>");
+    pub const DUFFLE_FILL: BI = BI("<i class=\"bi bi-duffle-fill\"></i>");
     pub const EAR: BI = BI("<i class=\"bi bi-ear\"></i>");
     pub const EAR_FILL: BI = BI("<i class=\"bi bi-ear-fill\"></i>");
     pub const EARBUDS: BI = BI("<i class=\"bi bi-earbuds\"></i>");
@@ -747,12 +776,18 @@
     pub const EJECT_FILL: BI = BI("<i class=\"bi bi-eject-fill\"></i>");
     pub const EMOJI_ANGRY: BI = BI("<i class=\"bi bi-emoji-angry\"></i>");
     pub const EMOJI_ANGRY_FILL: BI = BI("<i class=\"bi bi-emoji-angry-fill\"></i>");
+    pub const EMOJI_ASTONISHED: BI = BI("<i class=\"bi bi-emoji-astonished\"></i>");
+    pub const EMOJI_ASTONISHED_FILL: BI = BI("<i class=\"bi bi-emoji-astonished-fill\"></i>");
     pub const EMOJI_DIZZY: BI = BI("<i class=\"bi bi-emoji-dizzy\"></i>");
     pub const EMOJI_DIZZY_FILL: BI = BI("<i class=\"bi bi-emoji-dizzy-fill\"></i>");
     pub const EMOJI_EXPRESSIONLESS: BI = BI("<i class=\"bi bi-emoji-expressionless\"></i>");
     pub const EMOJI_EXPRESSIONLESS_FILL: BI = BI("<i class=\"bi bi-emoji-expressionless-fill\"></i>");
     pub const EMOJI_FROWN: BI = BI("<i class=\"bi bi-emoji-frown\"></i>");
     pub const EMOJI_FROWN_FILL: BI = BI("<i class=\"bi bi-emoji-frown-fill\"></i>");
+    pub const EMOJI_GRIMACE: BI = BI("<i class=\"bi bi-emoji-grimace\"></i>");
+    pub const EMOJI_GRIMACE_FILL: BI = BI("<i class=\"bi bi-emoji-grimace-fill\"></i>");
+    pub const EMOJI_GRIN: BI = BI("<i class=\"bi bi-emoji-grin\"></i>");
+    pub const EMOJI_GRIN_FILL: BI = BI("<i class=\"bi bi-emoji-grin-fill\"></i>");
     pub const EMOJI_HEART_EYES: BI = BI("<i class=\"bi bi-emoji-heart-eyes\"></i>");
     pub const EMOJI_HEART_EYES_FILL: BI = BI("<i class=\"bi bi-emoji-heart-eyes-fill\"></i>");
     pub const EMOJI_KISS: BI = BI("<i class=\"bi bi-emoji-kiss\"></i>");
@@ -767,9 +802,17 @@
     pub const EMOJI_SMILE_UPSIDE_DOWN_FILL: BI = BI("<i class=\"bi bi-emoji-smile-upside-down-fill\"></i>");
     pub const EMOJI_SUNGLASSES: BI = BI("<i class=\"bi bi-emoji-sunglasses\"></i>");
     pub const EMOJI_SUNGLASSES_FILL: BI = BI("<i class=\"bi bi-emoji-sunglasses-fill\"></i>");
+    pub const EMOJI_SURPRISE: BI = BI("<i class=\"bi bi-emoji-surprise\"></i>");
+    pub const EMOJI_SURPRISE_FILL: BI = BI("<i class=\"bi bi-emoji-surprise-fill\"></i>");
+    pub const EMOJI_TEAR: BI = BI("<i class=\"bi bi-emoji-tear\"></i>");
+    pub const EMOJI_TEAR_FILL: BI = BI("<i class=\"bi bi-emoji-tear-fill\"></i>");
     pub const EMOJI_WINK: BI = BI("<i class=\"bi bi-emoji-wink\"></i>");
     pub const EMOJI_WINK_FILL: BI = BI("<i class=\"bi bi-emoji-wink-fill\"></i>");
     pub const ENVELOPE: BI = BI("<i class=\"bi bi-envelope\"></i>");
+    pub const ENVELOPE_ARROW_DOWN: BI = BI("<i class=\"bi bi-envelope-arrow-down\"></i>");
+    pub const ENVELOPE_ARROW_DOWN_FILL: BI = BI("<i class=\"bi bi-envelope-arrow-down-fill\"></i>");
+    pub const ENVELOPE_ARROW_UP: BI = BI("<i class=\"bi bi-envelope-arrow-up\"></i>");
+    pub const ENVELOPE_ARROW_UP_FILL: BI = BI("<i class=\"bi bi-envelope-arrow-up-fill\"></i>");
     pub const ENVELOPE_AT: BI = BI("<i class=\"bi bi-envelope-at\"></i>");
     pub const ENVELOPE_AT_FILL: BI = BI("<i class=\"bi bi-envelope-at-fill\"></i>");
     pub const ENVELOPE_CHECK: BI = BI("<i class=\"bi bi-envelope-check\"></i>");
@@ -818,6 +861,7 @@
     pub const EXCLUDE: BI = BI("<i class=\"bi bi-exclude\"></i>");
     pub const EXPLICIT: BI = BI("<i class=\"bi bi-explicit\"></i>");
     pub const EXPLICIT_FILL: BI = BI("<i class=\"bi bi-explicit-fill\"></i>");
+    pub const EXPOSURE: BI = BI("<i class=\"bi bi-exposure\"></i>");
     pub const EYE: BI = BI("<i class=\"bi bi-eye\"></i>");
     pub const EYE_FILL: BI = BI("<i class=\"bi bi-eye-fill\"></i>");
     pub const EYE_SLASH: BI = BI("<i class=\"bi bi-eye-slash\"></i>");
@@ -832,6 +876,8 @@
     pub const FAST_FORWARD_CIRCLE: BI = BI("<i class=\"bi bi-fast-forward-circle\"></i>");
     pub const FAST_FORWARD_CIRCLE_FILL: BI = BI("<i class=\"bi bi-fast-forward-circle-fill\"></i>");
     pub const FAST_FORWARD_FILL: BI = BI("<i class=\"bi bi-fast-forward-fill\"></i>");
+    pub const FEATHER: BI = BI("<i class=\"bi bi-feather\"></i>");
+    pub const FEATHER2: BI = BI("<i class=\"bi bi-feather2\"></i>");
     pub const FILE: BI = BI("<i class=\"bi bi-file\"></i>");
     pub const FILE_ARROW_DOWN: BI = BI("<i class=\"bi bi-file-arrow-down\"></i>");
     pub const FILE_ARROW_DOWN_FILL: BI = BI("<i class=\"bi bi-file-arrow-down-fill\"></i>");
@@ -1023,6 +1069,10 @@
     pub const FIRE: BI = BI("<i class=\"bi bi-fire\"></i>");
     pub const FLAG: BI = BI("<i class=\"bi bi-flag\"></i>");
     pub const FLAG_FILL: BI = BI("<i class=\"bi bi-flag-fill\"></i>");
+    pub const FLOPPY: BI = BI("<i class=\"bi bi-floppy\"></i>");
+    pub const FLOPPY_FILL: BI = BI("<i class=\"bi bi-floppy-fill\"></i>");
+    pub const FLOPPY2: BI = BI("<i class=\"bi bi-floppy2\"></i>");
+    pub const FLOPPY2_FILL: BI = BI("<i class=\"bi bi-floppy2-fill\"></i>");
     pub const FLOWER1: BI = BI("<i class=\"bi bi-flower1\"></i>");
     pub const FLOWER2: BI = BI("<i class=\"bi bi-flower2\"></i>");
     pub const FLOWER3: BI = BI("<i class=\"bi bi-flower3\"></i>");
@@ -1056,6 +1106,7 @@
     pub const GENDER_AMBIGUOUS: BI = BI("<i class=\"bi bi-gender-ambiguous\"></i>");
     pub const GENDER_FEMALE: BI = BI("<i class=\"bi bi-gender-female\"></i>");
     pub const GENDER_MALE: BI = BI("<i class=\"bi bi-gender-male\"></i>");
+    pub const GENDER_NEUTER: BI = BI("<i class=\"bi bi-gender-neuter\"></i>");
     pub const GENDER_TRANS: BI = BI("<i class=\"bi bi-gender-trans\"></i>");
     pub const GEO: BI = BI("<i class=\"bi bi-geo\"></i>");
     pub const GEO_ALT: BI = BI("<i class=\"bi bi-geo-alt\"></i>");
@@ -1065,6 +1116,7 @@
     pub const GIFT_FILL: BI = BI("<i class=\"bi bi-gift-fill\"></i>");
     pub const GIT: BI = BI("<i class=\"bi bi-git\"></i>");
     pub const GITHUB: BI = BI("<i class=\"bi bi-github\"></i>");
+    pub const GITLAB: BI = BI("<i class=\"bi bi-gitlab\"></i>");
     pub const GLOBE: BI = BI("<i class=\"bi bi-globe\"></i>");
     pub const GLOBE_AMERICAS: BI = BI("<i class=\"bi bi-globe-americas\"></i>");
     pub const GLOBE_ASIA_AUSTRALIA: BI = BI("<i class=\"bi bi-globe-asia-australia\"></i>");
@@ -1134,6 +1186,8 @@
     pub const HEXAGON: BI = BI("<i class=\"bi bi-hexagon\"></i>");
     pub const HEXAGON_FILL: BI = BI("<i class=\"bi bi-hexagon-fill\"></i>");
     pub const HEXAGON_HALF: BI = BI("<i class=\"bi bi-hexagon-half\"></i>");
+    pub const HIGHLIGHTER: BI = BI("<i class=\"bi bi-highlighter\"></i>");
+    pub const HIGHLIGHTS: BI = BI("<i class=\"bi bi-highlights\"></i>");
     pub const HOSPITAL: BI = BI("<i class=\"bi bi-hospital\"></i>");
     pub const HOSPITAL_FILL: BI = BI("<i class=\"bi bi-hospital-fill\"></i>");
     pub const HOURGLASS: BI = BI("<i class=\"bi bi-hourglass\"></i>");
@@ -1262,17 +1316,22 @@
     pub const LIST_UL: BI = BI("<i class=\"bi bi-list-ul\"></i>");
     pub const LOCK: BI = BI("<i class=\"bi bi-lock\"></i>");
     pub const LOCK_FILL: BI = BI("<i class=\"bi bi-lock-fill\"></i>");
+    pub const LUGGAGE: BI = BI("<i class=\"bi bi-luggage\"></i>");
+    pub const LUGGAGE_FILL: BI = BI("<i class=\"bi bi-luggage-fill\"></i>");
     pub const LUNGS: BI = BI("<i class=\"bi bi-lungs\"></i>");
     pub const LUNGS_FILL: BI = BI("<i class=\"bi bi-lungs-fill\"></i>");
     pub const MAGIC: BI = BI("<i class=\"bi bi-magic\"></i>");
     pub const MAGNET: BI = BI("<i class=\"bi bi-magnet\"></i>");
     pub const MAGNET_FILL: BI = BI("<i class=\"bi bi-magnet-fill\"></i>");
     pub const MAILBOX: BI = BI("<i class=\"bi bi-mailbox\"></i>");
+    pub const MAILBOX_FLAG: BI = BI("<i class=\"bi bi-mailbox-flag\"></i>");
     pub const MAILBOX2: BI = BI("<i class=\"bi bi-mailbox2\"></i>");
+    pub const MAILBOX2_FLAG: BI = BI("<i class=\"bi bi-mailbox2-flag\"></i>");
     pub const MAP: BI = BI("<i class=\"bi bi-map\"></i>");
     pub const MAP_FILL: BI = BI("<i class=\"bi bi-map-fill\"></i>");
     pub const MARKDOWN: BI = BI("<i class=\"bi bi-markdown\"></i>");
     pub const MARKDOWN_FILL: BI = BI("<i class=\"bi bi-markdown-fill\"></i>");
+    pub const MARKER_TIP: BI = BI("<i class=\"bi bi-marker-tip\"></i>");
     pub const MASK: BI = BI("<i class=\"bi bi-mask\"></i>");
     pub const MASTODON: BI = BI("<i class=\"bi bi-mastodon\"></i>");
     pub const MEDIUM: BI = BI("<i class=\"bi bi-medium\"></i>");
@@ -1325,12 +1384,16 @@
     pub const NODE_MINUS_FILL: BI = BI("<i class=\"bi bi-node-minus-fill\"></i>");
     pub const NODE_PLUS: BI = BI("<i class=\"bi bi-node-plus\"></i>");
     pub const NODE_PLUS_FILL: BI = BI("<i class=\"bi bi-node-plus-fill\"></i>");
+    pub const NOISE_REDUCTION: BI = BI("<i class=\"bi bi-noise-reduction\"></i>");
     pub const NUT: BI = BI("<i class=\"bi bi-nut\"></i>");
     pub const NUT_FILL: BI = BI("<i class=\"bi bi-nut-fill\"></i>");
     pub const NVIDIA: BI = BI("<i class=\"bi bi-nvidia\"></i>");
+    pub const NVME: BI = BI("<i class=\"bi bi-nvme\"></i>");
+    pub const NVME_FILL: BI = BI("<i class=\"bi bi-nvme-fill\"></i>");
     pub const OCTAGON: BI = BI("<i class=\"bi bi-octagon\"></i>");
     pub const OCTAGON_FILL: BI = BI("<i class=\"bi bi-octagon-fill\"></i>");
     pub const OCTAGON_HALF: BI = BI("<i class=\"bi bi-octagon-half\"></i>");
+    pub const OPENCOLLECTIVE: BI = BI("<i class=\"bi bi-opencollective\"></i>");
     pub const OPTICAL_AUDIO: BI = BI("<i class=\"bi bi-optical-audio\"></i>");
     pub const OPTICAL_AUDIO_FILL: BI = BI("<i class=\"bi bi-optical-audio-fill\"></i>");
     pub const OPTION: BI = BI("<i class=\"bi bi-option\"></i>");
@@ -1347,6 +1410,8 @@
     pub const PARAGRAPH: BI = BI("<i class=\"bi bi-paragraph\"></i>");
     pub const PASS: BI = BI("<i class=\"bi bi-pass\"></i>");
     pub const PASS_FILL: BI = BI("<i class=\"bi bi-pass-fill\"></i>");
+    pub const PASSPORT: BI = BI("<i class=\"bi bi-passport\"></i>");
+    pub const PASSPORT_FILL: BI = BI("<i class=\"bi bi-passport-fill\"></i>");
     pub const PATCH_CHECK: BI = BI("<i class=\"bi bi-patch-check\"></i>");
     pub const PATCH_CHECK_FILL: BI = BI("<i class=\"bi bi-patch-check-fill\"></i>");
     pub const PATCH_EXCLAMATION: BI = BI("<i class=\"bi bi-patch-exclamation\"></i>");
@@ -1369,6 +1434,8 @@
     pub const PC_DISPLAY_HORIZONTAL: BI = BI("<i class=\"bi bi-pc-display-horizontal\"></i>");
     pub const PC_HORIZONTAL: BI = BI("<i class=\"bi bi-pc-horizontal\"></i>");
     pub const PCI_CARD: BI = BI("<i class=\"bi bi-pci-card\"></i>");
+    pub const PCI_CARD_NETWORK: BI = BI("<i class=\"bi bi-pci-card-network\"></i>");
+    pub const PCI_CARD_SOUND: BI = BI("<i class=\"bi bi-pci-card-sound\"></i>");
     pub const PEACE: BI = BI("<i class=\"bi bi-peace\"></i>");
     pub const PEACE_FILL: BI = BI("<i class=\"bi bi-peace-fill\"></i>");
     pub const PEN: BI = BI("<i class=\"bi bi-pen\"></i>");
@@ -1384,6 +1451,7 @@
     pub const PERCENT: BI = BI("<i class=\"bi bi-percent\"></i>");
     pub const PERSON: BI = BI("<i class=\"bi bi-person\"></i>");
     pub const PERSON_ADD: BI = BI("<i class=\"bi bi-person-add\"></i>");
+    pub const PERSON_ARMS_UP: BI = BI("<i class=\"bi bi-person-arms-up\"></i>");
     pub const PERSON_BADGE: BI = BI("<i class=\"bi bi-person-badge\"></i>");
     pub const PERSON_BADGE_FILL: BI = BI("<i class=\"bi bi-person-badge-fill\"></i>");
     pub const PERSON_BOUNDING_BOX: BI = BI("<i class=\"bi bi-person-bounding-box\"></i>");
@@ -1412,15 +1480,20 @@
     pub const PERSON_LOCK: BI = BI("<i class=\"bi bi-person-lock\"></i>");
     pub const PERSON_PLUS: BI = BI("<i class=\"bi bi-person-plus\"></i>");
     pub const PERSON_PLUS_FILL: BI = BI("<i class=\"bi bi-person-plus-fill\"></i>");
+    pub const PERSON_RAISED_HAND: BI = BI("<i class=\"bi bi-person-raised-hand\"></i>");
     pub const PERSON_ROLODEX: BI = BI("<i class=\"bi bi-person-rolodex\"></i>");
     pub const PERSON_SLASH: BI = BI("<i class=\"bi bi-person-slash\"></i>");
     pub const PERSON_SQUARE: BI = BI("<i class=\"bi bi-person-square\"></i>");
+    pub const PERSON_STANDING: BI = BI("<i class=\"bi bi-person-standing\"></i>");
+    pub const PERSON_STANDING_DRESS: BI = BI("<i class=\"bi bi-person-standing-dress\"></i>");
     pub const PERSON_UP: BI = BI("<i class=\"bi bi-person-up\"></i>");
     pub const PERSON_VCARD: BI = BI("<i class=\"bi bi-person-vcard\"></i>");
     pub const PERSON_VCARD_FILL: BI = BI("<i class=\"bi bi-person-vcard-fill\"></i>");
     pub const PERSON_VIDEO: BI = BI("<i class=\"bi bi-person-video\"></i>");
     pub const PERSON_VIDEO2: BI = BI("<i class=\"bi bi-person-video2\"></i>");
     pub const PERSON_VIDEO3: BI = BI("<i class=\"bi bi-person-video3\"></i>");
+    pub const PERSON_WALKING: BI = BI("<i class=\"bi bi-person-walking\"></i>");
+    pub const PERSON_WHEELCHAIR: BI = BI("<i class=\"bi bi-person-wheelchair\"></i>");
     pub const PERSON_WORKSPACE: BI = BI("<i class=\"bi bi-person-workspace\"></i>");
     pub const PERSON_X: BI = BI("<i class=\"bi bi-person-x\"></i>");
     pub const PERSON_X_FILL: BI = BI("<i class=\"bi bi-person-x-fill\"></i>");
@@ -1498,6 +1571,7 @@
     pub const R_CIRCLE_FILL: BI = BI("<i class=\"bi bi-r-circle-fill\"></i>");
     pub const R_SQUARE: BI = BI("<i class=\"bi bi-r-square\"></i>");
     pub const R_SQUARE_FILL: BI = BI("<i class=\"bi bi-r-square-fill\"></i>");
+    pub const RADAR: BI = BI("<i class=\"bi bi-radar\"></i>");
     pub const RADIOACTIVE: BI = BI("<i class=\"bi bi-radioactive\"></i>");
     pub const RAINBOW: BI = BI("<i class=\"bi bi-rainbow\"></i>");
     pub const RECEIPT: BI = BI("<i class=\"bi bi-receipt\"></i>");
@@ -1558,6 +1632,10 @@
     pub const SEARCH_HEART_FILL: BI = BI("<i class=\"bi bi-search-heart-fill\"></i>");
     pub const SEGMENTED_NAV: BI = BI("<i class=\"bi bi-segmented-nav\"></i>");
     pub const SEND: BI = BI("<i class=\"bi bi-send\"></i>");
+    pub const SEND_ARROW_DOWN: BI = BI("<i class=\"bi bi-send-arrow-down\"></i>");
+    pub const SEND_ARROW_DOWN_FILL: BI = BI("<i class=\"bi bi-send-arrow-down-fill\"></i>");
+    pub const SEND_ARROW_UP: BI = BI("<i class=\"bi bi-send-arrow-up\"></i>");
+    pub const SEND_ARROW_UP_FILL: BI = BI("<i class=\"bi bi-send-arrow-up-fill\"></i>");
     pub const SEND_CHECK: BI = BI("<i class=\"bi bi-send-check\"></i>");
     pub const SEND_CHECK_FILL: BI = BI("<i class=\"bi bi-send-check-fill\"></i>");
     pub const SEND_DASH: BI = BI("<i class=\"bi bi-send-dash\"></i>");
@@ -1572,6 +1650,7 @@
     pub const SEND_X: BI = BI("<i class=\"bi bi-send-x\"></i>");
     pub const SEND_X_FILL: BI = BI("<i class=\"bi bi-send-x-fill\"></i>");
     pub const SERVER: BI = BI("<i class=\"bi bi-server\"></i>");
+    pub const SHADOWS: BI = BI("<i class=\"bi bi-shadows\"></i>");
     pub const SHARE: BI = BI("<i class=\"bi bi-share\"></i>");
     pub const SHARE_FILL: BI = BI("<i class=\"bi bi-share-fill\"></i>");
     pub const SHIELD: BI = BI("<i class=\"bi bi-shield\"></i>");
@@ -1643,6 +1722,8 @@
     pub const SIGNPOST_SPLIT_FILL: BI = BI("<i class=\"bi bi-signpost-split-fill\"></i>");
     pub const SIM: BI = BI("<i class=\"bi bi-sim\"></i>");
     pub const SIM_FILL: BI = BI("<i class=\"bi bi-sim-fill\"></i>");
+    pub const SIM_SLASH: BI = BI("<i class=\"bi bi-sim-slash\"></i>");
+    pub const SIM_SLASH_FILL: BI = BI("<i class=\"bi bi-sim-slash-fill\"></i>");
     pub const SINA_WEIBO: BI = BI("<i class=\"bi bi-sina-weibo\"></i>");
     pub const SKIP_BACKWARD: BI = BI("<i class=\"bi bi-skip-backward\"></i>");
     pub const SKIP_BACKWARD_BTN: BI = BI("<i class=\"bi bi-skip-backward-btn\"></i>");
@@ -1697,6 +1778,7 @@
     pub const SORT_UP: BI = BI("<i class=\"bi bi-sort-up\"></i>");
     pub const SORT_UP_ALT: BI = BI("<i class=\"bi bi-sort-up-alt\"></i>");
     pub const SOUNDWAVE: BI = BI("<i class=\"bi bi-soundwave\"></i>");
+    pub const SOURCEFORGE: BI = BI("<i class=\"bi bi-sourceforge\"></i>");
     pub const SPEAKER: BI = BI("<i class=\"bi bi-speaker\"></i>");
     pub const SPEAKER_FILL: BI = BI("<i class=\"bi bi-speaker-fill\"></i>");
     pub const SPEEDOMETER: BI = BI("<i class=\"bi bi-speedometer\"></i>");
@@ -1730,6 +1812,7 @@
     pub const STRAVA: BI = BI("<i class=\"bi bi-strava\"></i>");
     pub const STRIPE: BI = BI("<i class=\"bi bi-stripe\"></i>");
     pub const SUBSCRIPT: BI = BI("<i class=\"bi bi-subscript\"></i>");
+    pub const SUBSTACK: BI = BI("<i class=\"bi bi-substack\"></i>");
     pub const SUBTRACT: BI = BI("<i class=\"bi bi-subtract\"></i>");
     pub const SUIT_CLUB: BI = BI("<i class=\"bi bi-suit-club\"></i>");
     pub const SUIT_CLUB_FILL: BI = BI("<i class=\"bi bi-suit-club-fill\"></i>");
@@ -1739,6 +1822,12 @@
     pub const SUIT_HEART_FILL: BI = BI("<i class=\"bi bi-suit-heart-fill\"></i>");
     pub const SUIT_SPADE: BI = BI("<i class=\"bi bi-suit-spade\"></i>");
     pub const SUIT_SPADE_FILL: BI = BI("<i class=\"bi bi-suit-spade-fill\"></i>");
+    pub const SUITCASE: BI = BI("<i class=\"bi bi-suitcase\"></i>");
+    pub const SUITCASE_FILL: BI = BI("<i class=\"bi bi-suitcase-fill\"></i>");
+    pub const SUITCASE_LG: BI = BI("<i class=\"bi bi-suitcase-lg\"></i>");
+    pub const SUITCASE_LG_FILL: BI = BI("<i class=\"bi bi-suitcase-lg-fill\"></i>");
+    pub const SUITCASE2: BI = BI("<i class=\"bi bi-suitcase2\"></i>");
+    pub const SUITCASE2_FILL: BI = BI("<i class=\"bi bi-suitcase2-fill\"></i>");
     pub const SUN: BI = BI("<i class=\"bi bi-sun\"></i>");
     pub const SUN_FILL: BI = BI("<i class=\"bi bi-sun-fill\"></i>");
     pub const SUNGLASSES: BI = BI("<i class=\"bi bi-sunglasses\"></i>");
@@ -1798,6 +1887,8 @@
     pub const THERMOMETER_LOW: BI = BI("<i class=\"bi bi-thermometer-low\"></i>");
     pub const THERMOMETER_SNOW: BI = BI("<i class=\"bi bi-thermometer-snow\"></i>");
     pub const THERMOMETER_SUN: BI = BI("<i class=\"bi bi-thermometer-sun\"></i>");
+    pub const THREADS: BI = BI("<i class=\"bi bi-threads\"></i>");
+    pub const THREADS_FILL: BI = BI("<i class=\"bi bi-threads-fill\"></i>");
     pub const THREE_DOTS: BI = BI("<i class=\"bi bi-three-dots\"></i>");
     pub const THREE_DOTS_VERTICAL: BI = BI("<i class=\"bi bi-three-dots-vertical\"></i>");
     pub const THUNDERBOLT: BI = BI("<i class=\"bi bi-thunderbolt\"></i>");
@@ -1824,6 +1915,7 @@
     pub const TRAIN_LIGHTRAIL_FRONT: BI = BI("<i class=\"bi bi-train-lightrail-front\"></i>");
     pub const TRAIN_LIGHTRAIL_FRONT_FILL: BI = BI("<i class=\"bi bi-train-lightrail-front-fill\"></i>");
     pub const TRANSLATE: BI = BI("<i class=\"bi bi-translate\"></i>");
+    pub const TRANSPARENCY: BI = BI("<i class=\"bi bi-transparency\"></i>");
     pub const TRASH: BI = BI("<i class=\"bi bi-trash\"></i>");
     pub const TRASH_FILL: BI = BI("<i class=\"bi bi-trash-fill\"></i>");
     pub const TRASH2: BI = BI("<i class=\"bi bi-trash2\"></i>");
@@ -1848,11 +1940,15 @@
     pub const TV_FILL: BI = BI("<i class=\"bi bi-tv-fill\"></i>");
     pub const TWITCH: BI = BI("<i class=\"bi bi-twitch\"></i>");
     pub const TWITTER: BI = BI("<i class=\"bi bi-twitter\"></i>");
+    pub const TWITTER_X: BI = BI("<i class=\"bi bi-twitter-x\"></i>");
     pub const TYPE: BI = BI("<i class=\"bi bi-type\"></i>");
     pub const TYPE_BOLD: BI = BI("<i class=\"bi bi-type-bold\"></i>");
     pub const TYPE_H1: BI = BI("<i class=\"bi bi-type-h1\"></i>");
     pub const TYPE_H2: BI = BI("<i class=\"bi bi-type-h2\"></i>");
     pub const TYPE_H3: BI = BI("<i class=\"bi bi-type-h3\"></i>");
+    pub const TYPE_H4: BI = BI("<i class=\"bi bi-type-h4\"></i>");
+    pub const TYPE_H5: BI = BI("<i class=\"bi bi-type-h5\"></i>");
+    pub const TYPE_H6: BI = BI("<i class=\"bi bi-type-h6\"></i>");
     pub const TYPE_ITALIC: BI = BI("<i class=\"bi bi-type-italic\"></i>");
     pub const TYPE_STRIKETHROUGH: BI = BI("<i class=\"bi bi-type-strikethrough\"></i>");
     pub const TYPE_UNDERLINE: BI = BI("<i class=\"bi bi-type-underline\"></i>");
@@ -1891,6 +1987,7 @@
     pub const VECTOR_PEN: BI = BI("<i class=\"bi bi-vector-pen\"></i>");
     pub const VIEW_LIST: BI = BI("<i class=\"bi bi-view-list\"></i>");
     pub const VIEW_STACKED: BI = BI("<i class=\"bi bi-view-stacked\"></i>");
+    pub const VIGNETTE: BI = BI("<i class=\"bi bi-vignette\"></i>");
     pub const VIMEO: BI = BI("<i class=\"bi bi-vimeo\"></i>");
     pub const VINYL: BI = BI("<i class=\"bi bi-vinyl\"></i>");
     pub const VINYL_FILL: BI = BI("<i class=\"bi bi-vinyl-fill\"></i>");
```

isosphere commented on examples/icons/index.html:
```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8"/>
     <title>Yew App</title>
-    <link rel="stylesheet" href="bootstrap-icons-v1.10.5/bootstrap-icons.css"/>
+    <link rel="stylesheet" href="bootstrap-icons-v1.11.3/bootstrap-icons.css"/>
```
Perhaps we should not put the version in the directory; it is already in the header of the file.

isosphere replied:
I disagree with myself now re: caching

isosphere commented on packages/yew-bootstrap/bootstrap-icons-v1.11.3/bootstrap-icons.css:
```diff
@@ -1,14 +1,14 @@
 /*!
- * Bootstrap Icons v1.10.5 (https://icons.getbootstrap.com/)
- * Copyright 2019-2023 The Bootstrap Authors
+ * Bootstrap Icons v1.11.3 (https://icons.getbootstrap.com/)
```
The version is identified here, so no need to identify it in the directory and complicate things.

e: I'm wrong; there's value for caching systems

isosphere reviewed (changes requested):
Update looks good; but since every such update will be complicated by this path change - I think we might as well make this the last "breaking" update to icons by removing the reference to the version number in the directory.

alexkazik commented:
I agree that it's not a good idea to manually change the version in the path when upgrading this package.

The downside of removing the version from the path I see is that trunk does not apply the hash-renaming and thus a new version would be at the same path and a browser might think it's version is still accurate and thus not load it again.

I've added a commit which generates the link automatically.

If you prefer to remove the version from the path, I'll do that.

isosphere commented on examples/icons/src/bin/link-bootstrap-icons.rs:
```diff
@@ -0,0 +1,31 @@
+        );
+        for _ in 0..l {
+            path_base = path_base.join("..");
+        }
```
I'm having trouble understanding what this looping is for, could you please explain it?

alexkazik commented:
I've updated the branch with better code.
Also there is now an option to move all the generating code into the library.
Depending on what you prefer, I'll:
- remove the last two commits and remove the version from the directory
- remove the last commit (in the example)
- merge the last two commits (in the library)

isosphere commented on examples/icons/src/bin/link-bootstrap-icons.rs:
```diff
@@ -0,0 +1,8 @@
+
+fn main() -> Result<(), std::io::Error> {
+    BIFiles::generate_link_bootstrap_icons(&PathBuf::from(
+        std::env::var("TRUNK_SOURCE_DIR").expect("Environment variable TRUNK_SOURCE_DIR"),
```
Looks like this environment variable gets automatically set when running with trunk, cool beans. We might be screwing over people that use wasm-pack or something.

isosphere commented on packages/yew-bootstrap/src/icons/mod.rs:
```diff
@@ -262,6 +262,53 @@ impl BIFiles {
+            let mut path = std::path::PathBuf::from(base_path);
+            for _ in 0..l {
+                path.push("..");
+            }
```
I think this looping is going a level deeper until it finds the target directory. I don't love this experimental approach, crawling the user's file system. It won't always work either: for example, on my systems I use this in my `~/.cargo/config`:

```
[build]
target-dir = "/home/matt/.cargo/build"
```

Meaning that my target directory is always under that shared directory, regardless of where I compile something. This build script will never find it. 

Looks like this is a common problem: https://github.com/rust-lang/cargo/issues/9661

isosphere reviewed (changes requested):
Appologizes, I never submitted this review. I didn't realize that the comments were invisible until I did so. 

Double apologies in that this review is probably annoying for you, especially after this delay. Generalizing this solution seems to be a problem that others are having, as mentioned above.

alexkazik commented:
No worries about the delay, though I was beginning to wonder.

Good point about the target folder, and I didn't even know it existed.
(I didn't invent the idea, just copied it as I saw it matching.)

Unfortunately there is no option in trunk to include the output of a command into the html.

I think this means that I should remove the last commit (the move this into the lib).

And leaves us with a few options:
- keep the second commit and keep the "auto finding" in the examples, but add a few comments about the problem
- remove that commit also (only 1 left) and
- - keep it as is (requires to change the version in the index.html each icons update, for those who don't use the cdn functionality)
- - remove the version from the path, which may has the problem of caching, as the files would now always have the same name and the name
- - do something else, though I don't know what

isosphere commented:
How about we include the generated file with each release, and rather than make a build script for generating these files we instead have a github action for it. We can control the environment of the runner for that action (i.e.: we can ensure the target directory will be found). The action will reside in our source tree, so folks can steal it if they want to DIY the generation.

I agree with your concerns around caching; it's probably best to have the version in the filename.

alexkazik commented:
> How about we include the generated file with each release, and rather than make a build script for generating these files we instead have a github action for it. We can control the environment of the runner for that action (i.e.: we can ensure the target directory will be found). The action will reside in our source tree, so folks can steal it if they want to DIY the generation.

Oh, that sounds like an good idea. Though I'm not good at github actions, maybe you can write it?

I've removed the commits with my try from the PR.

isosphere commented:
> Oh, that sounds like an good idea. Though I'm not good at github actions, maybe you can write it?

I can try; I'm no expert at it either; I've implemented them a couple of times with instructions.

alexkazik commented:
I was thinking about another problem and found another solution for this problem.

I've added it to this branch: https://github.com/alexkazik/yew-bootstrap/commits/update-icons2/
(Also I moved your commit to the beginning)

isosphere commented:
`trunk serve` gives me a weird result for the `examples/icons` crate using https://github.com/alexkazik/yew-bootstrap/commit/693f0a13ed3b20d25b487ff4701b564e84bf8c20:

![image](https://github.com/isosphere/yew-bootstrap/assets/163370/04cfddf9-3a4d-4ddb-b5d8-85a832996ebb)

No 404 errors in the network trace:

![image](https://github.com/isosphere/yew-bootstrap/assets/163370/6795ba95-9af0-468e-97fd-7aa8765ad65a)

The hook did run:

```
2024-06-29T13:28:44.403601Z  INFO spawned hook cargo command_arguments=["run", "--bin", "include-bootstrap-icons"]
2024-06-29T13:28:44.403629Z  INFO spawning hook stage=PostBuild command=cargo
...
     Running `/home/matt/.cargo/build/debug/include-bootstrap-icons`
2024-06-29T13:29:01.310198Z  INFO finished hook cargo
2024-06-29T13:29:01.310256Z  INFO applying new distribution
2024-06-29T13:29:01.310948Z  INFO ✅ success
2024-06-29T13:29:01.311031Z  INFO 📡 serving static assets at -> /
```

alexkazik commented:
That's really weird.
<img width="830" alt="Screenshot 2024-06-29 at 16 01 33" src="https://github.com/isosphere/yew-bootstrap/assets/612453/fedd70ba-5ef6-4197-b976-62fa0d7c9dc2">

```
❯ trunk serve -v    
2024-06-29T14:00:07.387637Z  INFO 🚀 Starting trunk 0.20.2
2024-06-29T14:00:07.387947Z DEBUG Spawning update check
2024-06-29T14:00:07.388350Z DEBUG Performing update check
2024-06-29T14:00:07.388799Z DEBUG Disable error reporting: false
2024-06-29T14:00:07.388902Z DEBUG Time since last check: 17h23m22s70ms501µs
2024-06-29T14:00:07.388905Z DEBUG No refresh needed
2024-06-29T14:00:07.388891Z DEBUG Enforce version - actual: 0.20.2, required: *
2024-06-29T14:00:07.388921Z DEBUG Current: 0.20.2, Most recent: 0.20.2
2024-06-29T14:00:07.390438Z DEBUG Build cooldown: None
2024-06-29T14:00:07.390537Z  INFO 📦 starting build
2024-06-29T14:00:07.390711Z DEBUG spawning asset pipelines
2024-06-29T14:00:07.555498Z DEBUG building icons
2024-06-29T14:00:07.555511Z DEBUG cargo args args=["build", "--target=wasm32-unknown-unknown", "--manifest-path", "/Users/alx/Develop/Rust/yew-bootstrap/examples/icons/Cargo.toml", "--bin", "icons"]
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.08s
warning: the following packages contain code that will be rejected by a future version of Rust: wasm-bindgen v0.2.87
note: to see what the problems were, use the option `--future-incompat-report`, or run `cargo report future-incompatibilities --id 1`
2024-06-29T14:00:07.674113Z DEBUG fetching cargo artifacts
2024-06-29T14:00:07.751895Z DEBUG get: Getting tool app=WasmBindgen version=Some("0.2.87") offline=false client_options=HttpClientOptions { root_certificate: None, accept_invalid_certificates: false }
2024-06-29T14:00:07.761153Z DEBUG get:find_system: system version found for wasm-bindgen: 0.2.92 app=WasmBindgen version=Some("0.2.87") offline=false client_options=HttpClientOptions { root_certificate: None, accept_invalid_certificates: false } app=WasmBindgen
2024-06-29T14:00:07.761165Z  INFO get: tool version mismatch (required: 0.2.87, system: 0.2.92) app=WasmBindgen version=Some("0.2.87") offline=false client_options=HttpClientOptions { root_certificate: None, accept_invalid_certificates: false }
2024-06-29T14:00:07.761231Z DEBUG get: Using wasm-bindgen (0.2.87) from: /Users/alx/Library/Caches/dev.trunkrs.trunk/wasm-bindgen-0.2.87/wasm-bindgen app=WasmBindgen version=Some("0.2.87") offline=false client_options=HttpClientOptions { root_certificate: None, accept_invalid_certificates: false }
2024-06-29T14:00:07.761279Z DEBUG calling wasm-bindgen for icons
2024-06-29T14:00:07.761283Z DEBUG wasm-bindgen args args=["--target=web", "--out-dir=/Users/alx/.cargo/target/wasm-bindgen/debug", "--out-name=icons", "/Users/alx/.cargo/target/wasm32-unknown-unknown/debug/icons.wasm", "--no-typescript"]
2024-06-29T14:00:07.888368Z DEBUG copying generated wasm-bindgen artifacts
2024-06-29T14:00:07.888384Z DEBUG processing hash for /Users/alx/.cargo/target/wasm32-unknown-unknown/debug/icons.wasm
2024-06-29T14:00:07.893662Z DEBUG copying /Users/alx/.cargo/target/wasm-bindgen/debug/icons.js to /Users/alx/Develop/Rust/yew-bootstrap/examples/icons/dist/.stage/icons-e85d219f42bdb2ee.js
2024-06-29T14:00:07.893908Z DEBUG copying /Users/alx/.cargo/target/wasm-bindgen/debug/icons_bg.wasm to /Users/alx/Develop/Rust/yew-bootstrap/examples/icons/dist/.stage/icons-e85d219f42bdb2ee_bg.wasm
2024-06-29T14:00:07.894165Z DEBUG recording SRI record - type: Sha384. name: icons-e85d219f42bdb2ee.js, value: OutputDigest { integrity: Sha384, hash: "o+0e1T1xHh0iH19thsXTIVtMV6Q58Pid77EyOETmtpzHyzAUZxsAbzdIX5fmzN0m" }
2024-06-29T14:00:07.897193Z DEBUG recording SRI record - type: Sha384. name: icons-e85d219f42bdb2ee_bg.wasm, value: OutputDigest { integrity: Sha384, hash: "N/hVSpfR8n5SmTYdULdzL0c+aTvDL7ESKupdQKpHlCg0W8CcWT62+p7XrAfO6xYi" }
2024-06-29T14:00:07.897200Z DEBUG rust build complete
2024-06-29T14:00:07.897773Z  INFO spawned hook cargo command_arguments=["run", "--bin", "include-bootstrap-icons"]
2024-06-29T14:00:07.897788Z  INFO spawning hook stage=PostBuild command=cargo
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.08s
warning: the following packages contain code that will be rejected by a future version of Rust: wasm-bindgen v0.2.87
note: to see what the problems were, use the option `--future-incompat-report`, or run `cargo report future-incompatibilities --id 1`
     Running `/Users/alx/.cargo/target/debug/include-bootstrap-icons`
2024-06-29T14:00:08.184941Z  INFO finished hook cargo
2024-06-29T14:00:08.184987Z  INFO applying new distribution
2024-06-29T14:00:08.188623Z  INFO ✅ success
2024-06-29T14:00:08.188904Z  INFO 📡 serving static assets at -> /
2024-06-29T14:00:08.188974Z  INFO 📡 server listening at:
2024-06-29T14:00:08.188979Z  INFO     🏠 http://127.0.0.1:8080/
2024-06-29T14:00:08.188984Z  INFO     🏠 http://[::1]:8080/
```

btw. I now also moved my target directory to a global space.

isosphere commented:
My bad; my browser settings were set to ignore font selections other than what I had configured. Your new approach appears to work well!

alexkazik commented:
Good to hear. I would force push the new approach to this MR if that's ok.

isosphere commented:
> Good to hear. I would force push the new approach to this MR if that's ok.

Go for it; I'll kill the existing branch on my end to avoid a mess.

alexkazik commented:
Done. And I'm satisfied with this solution.

(Btw. I also tried to reduce the number of version numbers, but tried to use const str but that can't be used with concat, good idea with the macro.)

isosphere commented:
> Done. And I'm satisfied with this solution.

Great! I'll do a final check locally and then merge it. Thanks for your contribution, you've improved the way we handle icons. 

> (Btw. I also tried to reduce the number of version numbers, but tried to use const str but that can't be used with concat, good idea with the macro.)

It seemed like a maintainance nightmare, it had to die! I started with a `const str` too, took a few iterations before I arrived at that approach. I think I should brush up on macros, they can really cut down on repetition / boilerplate stuff.

alexkazik commented:
Glad to help - until next time (which should now be easier ;-)
